routes/projects/files/get-one.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * GET /projects/:username/:projectName/files/:pathToFile
+ * The :pathToFile parameter must be URI encoded
+ * Returns the contents of the given file in the form:
+ * {
+ *   type: <string> content-type,
+ *   path: <string>,
+ *   size: <integer>,
+ *   link: <string>
```

Comment:
I should probably add a little more documentation here. I think our plan was to provide a `link` to the file contents (maybe in s3) as opposed to adding it to the metadata
cc @ashleygwilliams